content/en/docs/reference/access-authn-authz/psp-to-pod-security-standards.md

```diff
@@ -0,0 +1,272 @@
+          <li><code>level</code> is anything</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td><code>runAsUser</code></td>
+      <td>Mutating & Validating</td>
+      <td>
+        <p><b>Baseline</b>: Anything</p>
+        <p><b>Restricted</b>: <code>rule</code> is <code>MustRunAsNonRoot</code></p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>runAsGroup</code></td>
+      <td>Mutating (MustRunAs) & Validating</td>
+      <td>
+        <i>No opinion</i>
+      </td>
+    </tr>
+    <tr>
+      <td><code>supplementalGroups</code></td>
+      <td>Mutating & Validating</td>
+      <td>
+        <i>No opinion</i>
+      </td>
+    </tr>
+    <tr>
+      <td><code>fsGroup</code></td>
+      <td>Mutating & Validating</td>
+      <td>
+        <i>No opinion</i>
+      </td>
+    </tr>
+    <tr>
+      <td><code>readOnlyRootFilesystem</code></td>
+      <td>Mutating & Validating</td>
+      <td>
+        <i>No opinion</i>
+      </td>
+    </tr>
+    <tr>
+      <td><code>defaultAllowPrivilegeEscalation</code></td>
+      <td>Mutating</td>
+      <td>
+        <i>No opinion (non-validating)</i>
+      </td>
+    </tr>
+    <tr>
+      <td><code>allowPrivilegeEscalation</code></td>
+      <td>Mutating & Validating</td>
+      <td>
+        <p><i>Only mutating if set to <code>false</code></i></p>
+        <p><b>Baseline</b>: No opinion</p>
+        <p><b>Restricted</b>: <code>false</code></p>
+      </td>
+    </tr>
+    <tr>
+      <td><code>allowedHostPaths</code></td>
+      <td>Validating</td>
+      <td><i>No opinion (volumes takes precedence)</i></td>
+    </tr>
+    <tr>
+      <td><code>allowedFlexVolumes</code></td>
+      <td>Validating</td>
+      <td><i>No opinion (volumes takes precedence)</i></td>
+    </tr>
+    <tr>
+      <td><code>allowedCSIDrivers</code></td>
+      <td>Validating</td>
+      <td><i>No opinion (volumes takes precedence)</i></td>
+    </tr>
+    <tr>
+      <td><code>allowedUnsafeSysctls</code></td>
+      <td>Validating</td>
+      <td><b>Baseline & Restricted</b>: undefined / nil / empty</td>
+    </tr>
+    <tr>
+      <td><code>forbiddenSysctls</code></td>
+      <td>Validating</td>
+      <td><i>No opinion</i></td>
+    </tr>
+    <tr>
+      <td><code>allowedProcMountTypes</code><br><i>(alpha feature)</i></td>
+      <td>Validating</td>
+      <td><b>Baseline & Restricted</b>: <code>["Default"]</code> OR undefined / nil / empty</td>
+    </tr>
+    <tr>
+      <td><code>runtimeClass</code><br><code>&nbsp;.defaultRuntimeClassName</code></td>
+      <td>Mutating</td>
+      <td><i>No opinion</i></td>
+    </tr>
+    <tr>
+      <td><code>runtimeClass</code><br><code>&nbsp;.allowedRuntimeClassNames</code></td>
+      <td>Validating</td>
+      <td><i>No opinion</i></td>
+    </tr>
+  </tbody>
+</table>
+
+## PodSecurityPolicy Annotations
+
+The [annotaions](/docs/concepts/overview/working-with-objects/annotations/) enumerated in this
```

Comment:
```suggestion
## PodSecurityPolicy annotations

The [annotations](/docs/concepts/overview/working-with-objects/annotations/) enumerated in this
```

I'm happy to see this merged as-is (we can log a good-first-issue about fixing it).